extensions/skyportal/static/js/components/Archive.jsx

```diff
@@ -528,6 +628,111 @@ const Archive = () => {
             </Card>
           </Grid>
         </Grid>
+        <Dialog
+          fullScreen={fullScreen}
+          open={saveDialogOpen}
+          onClose={handleSaveDialogClose}
+          aria-labelledby="responsive-dialog-title"
+        >
+          <form onSubmit={handleSubmitSaveForm(onSubmitSave)}>
+            <DialogTitle id="responsive-dialog-title">
+              Save selected data to Fritz
+            </DialogTitle>
+            <DialogContent dividers>
+              <DialogContentText>
+                Post photometry data to source:
+              </DialogContentText>
+              <FormControl required>
+                <Controller
+                  name="obj_id"
+                  color="primary"
+                  render={(props) => (
+                    <RadioGroup
+                      color="primary"
+                      /* eslint-disable-next-line react/jsx-props-no-spreading */
+                      {...props}
+                      onChange={(event) => {
+                        props.onChange(event);
+                        if (event.target.value === createNewSourceText) {
+                         setSaveNewSource(true);
+                        }
+                        else {
+                          setSaveNewSource(false);
```

Comment:
indentation here doesn't match line 657 -- maybe that one needs another space?

Reply:
Fixed